framework/wazuh/manager.py

```diff
@@ -50,13 +45,12 @@ def get_status():
 
 @expose_resources(actions=[f"{'cluster' if cluster_enabled else 'manager'}:read_config"],
                   resources=[f'node:id:{node_id}' if cluster_enabled else '*:*:*'])
-def ossec_log(type_log='all', category='all', months=3, offset=0, limit=common.database_limit, sort_by=None,
+def ossec_log(level='all', tag='all', offset=0, limit=common.database_limit, sort_by=None,
```

Comment:
Eliminate "all" default value from level and tag, if no filter is applied all logs will be fetched.